docs/modules/operation/pages/database-reports/database.adoc

```diff
@@ -73,157 +73,162 @@ In addition, create a new entry in the `$\{OPENNMS_HOME}/etc/jasper-reports.xml`
-                                                                                                                              `snmphyaddr`: String, the MAC address of the interface, e.g., `005e607e9e00`. May be null. +
+                                                                                                                              `snmpifname`: String, the interface name of the interface; for example, `en0`. May be null. +
+                                                                                                                              `snmpifdescr`: String, the description of the interface; for example, `en0`. May be null. +
+                                                                                                                              `snmphyaddr`: String, the MAC address of the interface; for example, `005e607e9e00`. May be null. +
                                                                                                                               As each input parameter may be null, not all of them can be null at the same time. At least one input parameter has to be defined. +
                                                                                                                               +
-                                                                                                                              For more details see <<ga-database-reports-interface-descriptor, Interface descriptor use>>.
+                                                                                                                              For more details, see <<ga-database-reports-interface-descriptor, Interface descriptor use>>.
 |===
 
 [[ga-database-reports-node-source-descriptor]]
 === Node source descriptor use
 
-A node is addressed by a `node source descriptor`.
-The `node source descriptor` references the node either via the `foreign source` and `foreign id` or by the `node id`.
+A node is addressed by a node source descriptor.
+The node source descriptor references the node either via the foreign source and foreign id or by the node id.
 
-If `store by foreign source` is enabled addressing the node only via `foreign source` and `foreign id` is possible.
+If `store by foreign source` is enabled, is is possible to address the node only via `foreign source` and `foreign id`.
 
-To make report creation easier, there is a helper method to create the `node source descriptor`.
+To make report creation easier, there is a helper method to create the node source descriptor.
 
-NOTE: For more information about `store by foreign source`, please see link:https://wiki.opennms.org/wiki/ForeignSource/foreignId_Data_Storage_How-To[our Wiki].
+NOTE: For more information about store by foreign source, see link:https://wiki.opennms.org/wiki/ForeignSource/foreignId_Data_Storage_How-To[our Wiki].
```

Comment:
Should not have wiki links - and the referenced wiki page has also been moved to https://opennms.discourse.group/t/storing-data-with-foreign-sources/2057, though the content of this is something that should exist somewhere in the docs as a best practice.

Reply:
Updated and created Jira: https://issues.opennms.org/browse/NMS-13482